tests/perf/conftest.py

```diff
@@ -32,6 +32,18 @@ def pytest_addoption(parser: pytest.Parser):
         default=5000,
         help="Number of masks for black box methods." "Defaults to 5000.",
     )
+    parser.addoption(
+        "--dataset-data-root",
```

Comment:
`--dataset-root` might be a shorter alternative.

Reply:
Updated, thank you